code/game/objects/items/devices/scanners.dm

```diff
@@ -446,7 +446,7 @@ GENE SCANNER
 			else
 				render_list += "<span class='notice ml-1'>Subject contains no reagents in their stomach.</span>\n"
 
-		if(M.reagents.addiction_list.len)
+		if(LAZYLEN(M.reagents.addiction_list))
```

Comment:
This is a valid use of LAZY, becase we don't want to tell the doctor that their patient is addicted to "   " and then move on. Just pointing it out for reference.